### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by guilhermelinosp**

### CODE REVIEW

**Review by LaCapitainerie — CHANGES_REQUESTED** [review]
You litteraly add a space and ask for a PR..
Rejected.

**Comment by gmehdiev** [comment]
This is a good pull request.
Confirmed.

**Review by gabriel-logan — COMMENTED** [review]
Nem fudendo...

**Comment by coldmayo** [comment]
Wow, thanks for solving such a pressing issue! We really needed help with this and you were the only dev in the whole world who knew how to fix it. You're the chosen one.

**Comment by brenoepics** [comment]
LGTM!

**Comment by MrsRina** [comment]
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk

**Comment by P4nda-K1ng** [comment]
Why are these emails almost always shitposts

File: README [hunk 1]
```diff
@@ -15,4 +15,4 @@ several of them using the reStructuredText markup notation.
 
 Please read the Documentation/process/changes.rst file, as it contains the
 requirements for building and running the kernel, and information about
-the problems which may result by upgrading your kernel.
+the problems which may result by upgrading your kernel.
```
Comment (LaCapitainerie) [inline README:18]: Just a fucking space

**Review by LaCapitainerie — CHANGES_REQUESTED** [review]
Delete this PR, mail anoy me and the PR is not funny

**Comment by Real-Glitch** [comment]
ban this guy

**Comment by xXDeonoXx** [comment]
you are a shame for us Brazilians lol

**Comment by guilhermelinosp** [comment]
> [xXDeonoXx](/xXDeonoXx)

cuida da sua vida aee pokedex

**Comment by xXDeonoXx** [comment]
cuidado gente, o juninho incompetente ficou brabo KKKKKKKKKKKKK

Em qua., 26 de jun. de 2024 às 15:21, Guilherme (лино) Lino <
***@***.***> escreveu:

**Comment by MrsRina** [comment]
Totally disrespectful with Linux development community.

**Pull request closed without merging**